hdf5.rb

```diff
@@ -1,14 +1,13 @@
 class Hdf5 < Formula
   desc "File format designed to store large amounts of data"
   homepage "http://www.hdfgroup.org/HDF5"
-  url "http://www.hdfgroup.org/ftp/HDF5/current/src/hdf5-1.8.15-patch1.tar.bz2"
-  version "1.8.15"
-  sha256 "a5afc630c4443547fff15e9637b5b10404adbed4c00206d89517d32d6668fb32"
+  url "http://www.hdfgroup.org/ftp/HDF5/current/src/hdf5-1.8.16.tar.bz2"
```

Comment:
let's use `http://www.hdfgroup.org/ftp/HDF5/releases/hdf5-1.8.16/src/hdf5-1.8.16.tar.bz2` so that it does not break next time `hdf5` is updated.